package.json

```diff
@@ -301,5 +301,5 @@
       "current node"
     ]
   },
-  "packageManager": "pnpm@10.10.0"
+  "packageManager": "pnpm@10.17.1"
```

Comment:
Why did this change?

Reply:
Oops, i'll revert.
I did a `pnpm install` and pnpm said `run pnpm self-update`